views/js/pciCreator/dev/mathEntryInteraction/creator/widget/states/Question.js

```diff
@@ -0,0 +1,100 @@
+define([
+    'taoQtiItem/qtiCreator/widgets/states/factory',
+    'taoQtiItem/qtiCreator/widgets/interactions/states/Question',
+    'taoQtiItem/qtiCreator/widgets/helpers/formElement',
+    'taoQtiItem/qtiCreator/editor/simpleContentEditableElement',
+    'taoQtiItem/qtiCreator/editor/containerEditor',
+    'tpl!mathEntryInteraction/creator/tpl/propertiesForm'
+], function(stateFactory, Question, formElement, simpleEditor, containerEditor, formTpl){
+    'use strict';
+
+    var MathEntryInteractionStateQuestion = stateFactory.extend(Question, function(){
+
+        var $container = this.widget.$container,
+            $prompt = $container.find('.prompt'),
+            interaction = this.widget.element;
+
+        containerEditor.create($prompt, {
+            change : function(text){
+                interaction.data('prompt', text);
+                interaction.updateMarkup();
+            },
+            markup : interaction.markup,
+            markupSelector : '.prompt',
+            related : interaction
+        });
+
+    }, function(){
+        var $container = this.widget.$container,
+            $prompt = $container.find('.prompt');
+
+        simpleEditor.destroy($container);
+        containerEditor.destroy($prompt);
+    });
+
+    function configChangeCallBack(interaction, value, name) {
+        interaction.prop(name, value);
+        interaction.triggerPci('configChange', [interaction.getProperties()]);
+    }
+
+    MathEntryInteractionStateQuestion.prototype.initForm = function(){
```

Comment:
I think we agreed to name the function now... it is true that older PCIs do not alwalys apply this rule but if let's makes new ones better :)